tutorials/tts/MixerTTS_FastPitch_Training.ipynb

```diff
@@ -0,0 +1,704 @@
+    "                                                                               sup_data_types, \\\n",
+    "                                                                               text_tokenizer, \\\n",
+    "                                                                               text_normalizer, \\\n",
+    "                                                                               text_normalizer_call_kwargs)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "452305cd",
+   "metadata": {},
+   "source": [
+    "### FastPitch\n",
+    "\n",
+    "Now that we looked at the FastPitch model, let's see how to prepare all data for training it. \n",
+    "\n",
+    "Firstly, let's download all necessary training scripts and configs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "11bc1aca",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!wget https://raw.githubusercontent.com/NVIDIA/NeMo/main/examples/tts/fastpitch.py\n",
+    "\n",
+    "!mkdir -p conf && cd conf \\\n",
+    "&& wget https://raw.githubusercontent.com/NVIDIA/NeMo/main/examples/tts/conf/fastpitch_align_v1.05.yaml \\\n",
+    "&& cd .."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0a416486",
+   "metadata": {},
+   "source": [
+    "We will be using `nemo.collections.tts.torch.data.TTSDataset` for the entire data pipeline. \n",
+    "\n",
+    "In the Mixer-TTS pipeline we used a phoneme based tokenizer but in the FastPitch training pipeline we would like to demonstrate a character based tokenizer `nemo.collections.tts.torch.tts_tokenizers.EnglishCharsTokenizer`. Unlike Phoneme based tokenizer, `EnglishCharsTokenizer` does not need a phoneme dictionary or a heteronym dictionary. We will be using the same `nemo_text_processing.text_normalization.normalize.Normalizer` for normalizing the text as used in the Mixer-TTS example.\n",
+    "\n",
+    "Since this process does not need any additional data we just download the raw waveform audio files and the manifests."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "75cfe0e1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# download data and manifests\n",
+    "!wget https://github.com/NVIDIA/NeMo/releases/download/v0.11.0/test_data.tar.gz && mkdir -p tests/data && tar xzf test_data.tar.gz -C tests/data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "448a8e94",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!mkdir -p tts_dataset_files && cd tts_dataset_files \\\n",
+    "&& wget https://raw.githubusercontent.com/NVIDIA/NeMo/main/nemo_text_processing/text_normalization/en/data/whitelist_lj_speech.tsv \\\n",
+    "&& cd ..    "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4ef5bdb7",
+   "metadata": {},
+   "source": [
+    "Let's define the above mentioned Normalizer and Tokenizer"
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
You don't need to import it again, it was already in Mixer-TTS part. Remove it.

Reply:
Done.